www/docs/client/links/loggerLink.md

```diff
@@ -50,6 +50,11 @@ type LoggerLinkOptions<TRouter extends AnyRouter> = {
    * Used in the built-in defaultLogger
    */
   console?: ConsoleEsque;
+  /**
+   * Color mode used in the default logger.
+   * @default 'css'
+   */
+  colorMode?: 'ansi' | 'css';
```

Comment:
Could we do `ansi` the default if `typeof window === 'undefined`? 🤔

Reply:
sounds reasonable